src/frontend/apps/bbb/DashboardMeeting/index.tsx

```diff
@@ -64,14 +65,17 @@ const DashboardMeeting = () => {
   }`;
 
   useEffect(() => {
-    const attendeeFound = meeting?.infos?.attendees.find((attendee) => {
-      return (
-        consumerSiteUserId === attendee.userID &&
-        (attendee.hasVideo === 'true' ||
-          attendee.hasJoinedVoice === 'true' ||
-          attendee.isListeningOnly === 'true')
-      );
-    });
+    let attendeeFound: Attendee | undefined;
```

Comment:
```suggestion
    let attendeeFound: Maybe<Attendee>;
```

Reply:
And it's more `Nullable` no ? Because you are using `null` and not `undefined` in your type.